packages/functional-tests/tests/settings/recoveryKey.spec.ts

```diff
@@ -5,7 +5,6 @@
 import fs from 'fs';
 import pdfParse from 'pdf-parse';
 import { TestAccountTracker } from '../../lib/testAccountTracker';
-import { EmailHeader, EmailType } from '../../lib/email';
```

Comment:
I assume these were just unused and the linter was crying?

Reply:
Yep!